structuretoolkit/build/aimsgb.py

```diff
@@ -45,43 +47,57 @@ def grainboundary(
     sigma,
     plane,
     initial_struct,
-    to_primitive=False,
-    delete_layer="0b0t0b0t",
-    add_if_dist=0.0,
     uc_a=1,
     uc_b=1,
+    vacuum=0.0,
+    gap=0.0,
+    delete_layer="0b0t0b0t",
+    tol=0.25,
+    to_primitive=False,
+    add_if_dist=None,
```

Comment:
Not necessary, but beneficial.

Previously the arguments were `GrainBoundary.__init__` args, then an arbitrary mixture of `GrainBoudary.__init__` and `GrainBoundary.build_gb` kwargs. This rationalizes things so that it's `GrainBoundary.__init__` args, `GrainBoundary.__init__` kwargs, `Grain.stack_grains` kwargs, then deprecated kwargs, with user-facing kwargs appearing in the same order as they do in the underlying library.

Since we need to change the signature anyways to handle the new kwarg and the renamed kwarg, I thought we might as well tighten the API up at the same time.